daemon/pipeline-dbus-impl.cc

```diff
@@ -68,6 +69,24 @@ static void _pipeline_free (gpointer data)
   g_free (p);
 }
 
+/**
+ * @brief Get the skeleton object of the DBus interface.
+ */
+MachinelearningServicePipeline *
+gdbus_get_instance_pipeline (void)
```

Comment:
line 76 and 85, it would be a static.
Also, plz consider changing function name (gdbus_get_pipeline_instance / gdbus_put_pipeline_instance) same rule in model-dbus)

Reply:
I renamed those functions, PTAL